frontend/web/src/app/services/physical-print/physical-print-mock.service.ts

```diff
@@ -0,0 +1,32 @@
+import {Injectable} from "@angular/core";
+import {Observable} from "rxjs";
+import {PhysicalPart} from "../../model/physical-part";
+import {PhysicalPrint} from "../../model/physical-print";
+
+@Injectable()
+export class PhysicalPartMockService {
+
+  private data: PhysicalPrint[] = [
+    new PhysicalPrint({
+      "id": 1,
+      "name": "Cool print",
+    }), new PhysicalPrint({
+      "id": 2,
+      "name": "Fantastic print",
+    })
+  ];
+
+  constructor() {
+  }
+
+  getAll(): Observable<PhysicalPrint[]> {
+    return Observable.of(this.data)
+  }
+  get(id: number): Observable<PhysicalPrint> {
+    return Observable.of( this.data.filter(e => {if(e.id == id)return e})[0]);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') == should be === [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=typescript%3AS1440)